api/envoy/extensions/filters/network/dubbo_proxy/v3/dubbo_proxy.proto

```diff
@@ -32,7 +35,18 @@ enum SerializationType {
   Hessian2 = 0;
 }
 
-// [#next-free-field: 6]
+message Drds {
+  // Configuration source specifier.
+  // In case of *api_config_source* only aggregated *api_type* is supported.
+  config.core.v3.ConfigSource config_source = 1 [(validate.rules).message = {required: true}];
+
+  // The name of the route configuration. This allows to use different route
+  // configurations. Tells which route configuration should be fetched from the configuration source.
+  // Leave unspecified is also valid and means the unnamed route configuration.
+  string route_config_name = 2;
```

Comment:
nit: Because the templates we can't change the field name but in the comment you could mention this is the name of a MultipleRoute Configuration, not a RouteConfiguration.